conf/evolutions/default/92.sql

```diff
@@ -1,11 +1,15 @@
  # --- !Ups
-CREATE TABLE user_action_log (
-  id         SERIAL PRIMARY KEY,
-  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
-  user_id    INT REFERENCES users,
-  address    INET NOT NULL,
-  action     TEXT
-);
+ CREATE TABLE user_action_log (
+   id         SERIAL PRIMARY KEY,
+   created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+   user_id    INT REFERENCES users,
+   address    INET NOT NULL,
+   context    VARCHAR(40) NOT NULL,
+   context_id INT,
+   action     VARCHAR(40) NOT NULL,
```

Comment:
Why is this only 40 chars?

Reply:
Because it's only a name, it shouldn't be longer than that